driver/src/price_finding/naive_solver.rs

```diff
@@ -145,6 +188,30 @@ impl PriceFinding for NaiveSolver {
     }
 }
 
+fn u128_to_u256(x: u128) -> U256 {
```

Comment:
any reason for moving this? Do helper functions generally go below or above in Rust?

Reply:
I wanted to put them together, and since they are private I felt like they shouldn't be at the top of the file. Usually when I jump to a location I want to see the public structs and traits that I might use outside of this model, and not some model specific private helper method.

But not sure if this is general style.